Detectors/gconfig/g4config.in

```diff
@@ -7,7 +7,7 @@
 /mcTracking/loopVerbose 1
 /mcVerbose/composedPhysicsList 2
 /mcVerbose/runAction 2   # For looping thresholds control
-#/tracking/verbose 1
+/tracking/verbose 1
```

Comment:
```suggestion
#/tracking/verbose 1
```

Reply:
Hi, this is a remnants of the debugging session, you can remove it by accepting this change